app/objects/c_operation.py

```diff
@@ -245,35 +247,32 @@ def report(self, file_svc, output=False, redacted=False):
 
     """ PRIVATE """
 
-    async def _run_phases(self, planner):
-        for phase in self.adversary.phases:
+    async def _run(self, planner):
+        # The following is scope manipulation to force multiple loops of increasing sets of steps for atomic
+        # while forcing a single loop for a non-atomic
+        handle = self.adversary.atomic_ordering
+        if not self.atomic_enabled:
```

Comment:
I think these three lines (253-255) can be consolidated into a single line and look cleaner